docs/docs/topics/event-actions.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Event Rules and Actions
+---
+
+The event framework allows you to respond to [events](/topics/events) as they occur within Infrahub by defining filter conditions and triggering actions.
+By creating multiple rules and actions, you can build automated workflows.
+
+## How it works
+
+1. **Define trigger rules** that match specific events
+2. **Set filter conditions** to narrow down which events should trigger actions
+3. **Configure actions** to execute when conditions are met
+
+## Example workflows
+
+### Adding devices to active group
+
+**Workflow:**
+
+1. Device is updated → status changed to "active"
+2. Node trigger rule detects the change
+3. Group action adds device to "active-devices" group
+4. Group member added event is generated
+5. Generator action runs for the new member
+
+**Implementation:**
+
+1. **Node Trigger Rule**
+   * Matches `infrahub.node.updated` events when a device's `status` changes to "active"
+   * Executes a group action to add the device to the "active-devices" group
+
+2. **Group Trigger Rule**
+   * Matches `infrahub.group.member_added` events for the "active-devices" group
+   * Runs a generator definition targeting the newly added member
+
+### Removing devices from active group
+
+**Workflow:**
+
+1. Device is updated → status changed from "active"
+2. Node trigger rule detects the change
+3. Group action removes device from "active-devices" group
+
+## Trigger rules
+
+Trigger rules are nodes that inherit from the `CoreTriggerRule` generic and match against specific event types:
+
+| Rule Type | Description | Use Case |
+|-----------|-------------|----------|
+| `CoreNodeTriggerRule` | Filters on mutations to specific nodes | Match changes to attributes or relationships |
+| `CoreGroupTriggerRule` | Filters on changes to specific groups | Match when members are added or removed |
+
+### Configuring node trigger rule
+
+When setting up a node trigger rule:
+
+1. Select the node kind to match against
+2. Create match filters for specific attributes or relationships
+3. Set the trigger to active when configuration is complete
+
+#### Match filters
+
+::: info
```

Comment:
I think the space before info is causing some issues here.

![Screenshot 2025-06-11 at 12 58 53](https://github.com/user-attachments/assets/ceaab3c3-b0d3-4883-b70a-997055ad3b7f)